README.md

```diff
@@ -5,17 +5,34 @@ Minio is an object storage server released under Apache License v2.0. It is comp
 Minio server is light enough to be bundled with the application stack, similar to NodeJS, Redis and MySQL.
 
 ## Docker Container
+
 ### Stable
+
+```sh
+docker run -p 9000:9000 --name minio -d minio/minio server /export
+```
```

Comment:
That is true but verbosity here is large and shadowing others making it inconsistent.. That is why it is not needed to add options like `--name` and `-d` . They can be in more explanatory section and that can be referenced here to proceed for further reading. 

Just to get this running all they need is below to have Minio running. 

```
docker run -p 9000:9000 minio/minio server /export
``` 

Even with the explanation it is still incomplete we need to explain about how to have persistent access keys etc using environment variables or being able to persist `config.json` etc.